packages/@react-spectrum/dialog/stories/Dialog.stories.tsx

```diff
@@ -234,7 +234,77 @@ storiesOf('Dialog/Alert', module)
       onCancel: action('cancel'),
       autoFocusButton: 'cancel'
     })
+  )
+  .add(
+    'allowsKeyboardConfirmation',
+    () => renderAlert({
+      variant: 'error',
+      title: 'Error: Danger Will Robinson',
+      children: singleParagraph(),
+      primaryActionLabel: 'Accept',
+      cancelLabel: 'Cancel',
+      secondaryActionLabel: 'Secondary button',
+      onPrimaryAction: action('primary'),
+      onSecondaryAction: action('secondary'),
+      onCancel: action('cancel'),
+      allowsKeyboardConfirmation: true
+    })
+  )
+  .add(
+    'alert with content actions',
+    () => renderWithContentButtons({
+      variant: 'error',
+      title: 'Error: Danger Will Robinson',
+      primaryActionLabel: 'Accept',
+      cancelLabel: 'Cancel',
+      onPrimaryAction: action('primary'),
+      onSecondaryAction: action('secondary'),
+      onCancel: action('cancel'),
+      allowsKeyboardConfirmation: true
+    })
```

Comment:
```suggestion
    }),
    {description: {data: 'Pressing enter when focused on the counter button should not keyboard confirm the dialog.'}}
  
```

Reply:
great, I'll add these details and remove the docs section in an incoming commit. thanks a lot!